spec/data_utils_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'spec_helper'
+require_relative '../dependency'
+
+RSpec.describe DataUtils do
+  include described_class
+  let(:path) { 'rspec_test_data.yaml' }
+
+  summary = {
+    name: 'Rspec',
+    difficulty: 'Easy',
+    att_total: 1,
+    att_used: 1,
+    hints_total: 1,
+    hints_used: 1
+  }
+
+  describe '#save' do
+    it 'saves a TableRow object to a new file' do
+      File.new(path, 'a') unless File.exist?(path)
```

Comment:
Please change logic of your test to the following one
- generate the random file name, save it
- test your method
- delete the recently created file